scripts/mypyc

```diff
@@ -27,29 +30,128 @@ from mypyc.build import mypycify
     with open(setup_file, 'w') as f:
-        f.write(setup_format.format(sys.argv[1:], opt_level))
-
+        f.write(setup_format.format(paths, opt_level))
     # We don't use run_setup (like we do in the test suite) because it throws
     # away the error code from distutils, and we don't care about the slight
     # performance loss here.
     env = os.environ.copy()
     base_path = os.path.join(os.path.dirname(__file__), '..')
     env['PYTHONPATH'] = base_path + os.pathsep + env.get('PYTHONPATH', '')
     cmd = subprocess.run([sys.executable, setup_file, 'build_ext', '--inplace'], env=env)
+    return cmd.returncode
+
+
+def mypyc_build(options: Options) -> None:
+    if len(options.files) < 1:
+        sys.exit("no source files provided")
+    returncode = mypyc_compile(options.build_dir, options.files, options.opt_level)
+    sys.exit(returncode)
+
+
+def mypyc_run(options: Options) -> None:
+    if len(options.files) < 1:
+        sys.exit("no source file provided")
+    returncode = mypyc_compile(options.build_dir, options.files[0:1], options.opt_level)
+    if returncode != 0:
+        sys.exit(returncode)
+    module_name = os.path.basename(options.files[0])
+    module_name = os.path.splitext(module_name)[0]
+    import_command = '"import {}"'.format(module_name)
+    # TODO: well this is dumb, we'd come up with a better way
+    env = os.environ.copy()
+    base_path = os.path.join(os.path.dirname(__file__), '..')
+    env['PYTHONPATH'] = base_path + os.pathsep + env.get('PYTHONPATH', '')
+    # TODO: How do we guarantee that we run the compiled version when both .py and .so files
+    # are in the same directory?
+    cmd = subprocess.run("python -c " + import_command, shell=True)
+    # TODO: I don't know why this is not working...
+    # cmd = subprocess.run([sys.executable, 'cmd', import_command], env=env, capture_output=True)
     sys.exit(cmd.returncode)
 
+
+def mypyc_clean(options: Options) -> None:
+    if os.path.exists(options.build_dir):
+        files = []
+        directories = []
+        # TODO: this simply hardcodes what generates now
+        # a better solution is to use some configuration files
+        # maybe use a `.mypyc_cache` folder to store them
+        with os.scandir(options.build_dir) as entries:  # type: ignore
+            for entry in entries:
+                if entry.is_dir() and entry.path.find("temp") != -1:
+                    directories.append(entry.path)
+                if entry.is_file() and (entry.path.endswith('.c') or entry.path.endswith('.h')
+                                        or entry.path.find('ops.txt') != -1):
+                    files.append(entry.path)
+        for file in files:
+            try:
+                os.remove(file)
+            except OSError as e:
+                print('Error "{}" occurred when removing {}'.format(e.strerror, file))
+        for directory in directories:
+            shutil.rmtree(directory)
```

Comment:
Can you also delete the `build` directory, if it's empty now? Also, we should delete the shared libraries outside the build directory (but somehow only those what have been compiled with mypyc using this build directory).